### PR TITLE
joomla: Small fixes in README

### PR DESCRIPTION
**Pull request by cimomo**
**What this PR does / why we need it**:
Typo and other small fixes.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/joomla/README.md [hunk 1]
```diff
@@ -63,8 +63,8 @@ The following table lists the configurable parameters of the Joomla! chart and t
 | `externalDatabase.port`           | Port of the external database          | `3306`                                                    |
 | `externalDatabase.user`           | Existing username in the external db   | `bn_joomla`                                               |
 | `externalDatabase.password`       | Password for the above username        | `nil`                                                     |
-| `externalDatabase.database`       | Name of the existing databse           | `bitnami_joomla`                                          |
-| `mariadb.enabled`                 | Wheter to use or not the mariadb chart | `true`                                                    |
+| `externalDatabase.database`       | Name of the existing database          | `bitnami_joomla`                                          |
+| `mariadb.enabled`                 | Whether to the MariaDB chart           | `true`                                                    |
```
Comment (javsalgar) [inline stable/joomla/README.md]: "Whether to the MariaDB chart" sounds strange to me. I am missing a verb

**Review by javsalgar — CHANGES_REQUESTED** [review]

File: stable/joomla/README.md (hunk 1 above)
Comment (cimomo) [reply stable/joomla/README.md]: My bad ... fat-fingered the change.

Comment (cimomo) [reply stable/joomla/README.md]: Updated the PR. Please take a look. @javsalgar

**Comment by timstoop** [comment]
/lgtm
thanks @javsalgar, I missed that one

**Review by javsalgar — APPROVED** [review]
Now LGTM

**Pull request merged**